panel/src/config/components.js

```diff
@@ -108,6 +108,7 @@ import Panel from "@/components/Layout/Panel.vue";
 import Stats from "@/components/Layout/Stats.vue";
 import Table from "@/components/Layout/Table.vue";
 import TableCell from "@/components/Layout/TableCell.vue";
+import TableUpdateStatusCell from "@/components/Layout/TableUpdateStatusCell.vue";
```

Comment:
This should probably rather go into `Forms/Previews` like other table cell previews.

Reply:
It's not a field preview though, is it? `TableCell.vue` supports either table cell components or field preview components. I used the former here.